### ISSUE TITLE
Markup control buttons as a list

### ISSUE THREAD

**Issue by kb-smarterbalanced** [ableplayer/ableplayer#506]
From a360 audit - https://accessible360.com/

The control buttons are in div element. These are semantically lists, but they are not marked up as such. Use list to markup elements.

**Comment by terrill** [comment]
@kb-smarterbalanced I agree with this in principle, but given the complexity of some of the components, I question whether it would really help. For buttons that trigger pop-up menus (captions, chapters, preferences, and volume) the result would be nested lists, and the preferences menu further triggers dialogs, which adds yet another layer.  If the seekbar is included in the list of controls (as it arguably should be, especially within the legacy skin), that's an extremely complex widget.

I'll continue to consider this, but I think it needs some A-B usability testing with screen reader users before we implement it.

**Comment by kb-smarterbalanced** [comment]
Sounds good, I will run it by my UX team to see if we want to do the testing.

**Comment by kb-smarterbalanced** [comment]
Hi Terrill, we have another issue that has the same recommendation from A360, we're going to change our code and test on a screen reader and we'll let you know how it goes. 
I agree with you though that nested lists really aren't a good idea in this case.

**Comment by joedolson** [comment]
I think that this is not a case where the use of a list would be effective. The groups of buttons are truly not list groups. I do think that there could be logic to using groups in some way, but a list role doesn't seem right to me - it would add semantic noise without creating any benefit.
